### DISCUSSION TITLE
Update 20.1-0.18-v2

### DISCUSSION THREAD

**Discussion by BeansGalaxy** [BeansGalaxy/Beans-Backpacks-2#87] [Announcements]
What are your thoughts on the new cauldrons? If it happens to be negative take a look at the adorable texture and re-consider your opinion before posting. 

Also how do you feel about the Decorated Pots receiving a cap to the item limit? I tried to balance this by reducing the weight of Enchanted Books in backpacks so you can carry more and 8,192 items is a lot of items but if you were moving more, what item were you moving? 

<details><summary>This might be the last big update before the full release so please, any feedback is welcome. :) </summary>
<p><img align="right" src="https://github.com/BeansGalaxy/Beans-Backpacks-2/blob/20.1+FluidStorage/assets/images/updates/wet_update.png?raw=true" alt="The Wet Update" style="width:400px"></p>
</details>

**Comment by bic060** [comment]
Hi, I'm currently on server with Cursed Walking 1.20.1 v1.3 modpack with beansbackpack 20.1-0.18-v2.
I got crashed whenever I equipped an ender backpack on my body and constantly crashing after. However, this only applied on ender backpack, all the remain backpacks are fine.
[crash-2024-03-18_17.34.16-client.txt](https://github.com/BeansGalaxy/Beans-Backpacks-2/files/14634724/crash-2024-03-18_17.34.16-client.txt)

**Reply by BeansGalaxy** [reply]
Fixed in 20.1-0.19-v2
